### PR TITLE
[CNSMR-3232]added external libraries policy

### PR DESCRIPTION
**Pull request by viorel15**
Ticket: https://babylonpartners.atlassian.net/browse/CNSMR-3232

This will replace the ticket from main repo: https://github.com/babylonhealth/babylon-ios/pull/10028

Why?

 The team needs a policy around adding external libraries to our project

### CODE REVIEW

File: Cookbook/Technical-Documents/External-libraries.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# External libraries policy
+
+
+If you are looking to add a new service in our app or adding an external framework to improve our code and tooling, then the procedure below must be followed.
+1. [Exploration](#exploration)
+1. [Implementation](#implementation)
+1. [Release](#release)
+
+## Exploration
+Even if the decision of using this framework has been already taken (for example this is a service that must adopted by your squad for all mobile platforms) this step is still mandatory.
```
Comment (konrad-em) [inline Cookbook/Technical-Documents/External-libraries.md]:
```suggestion
Even if the decision of using this framework has been already taken (for example this is a service that must be adopted by your squad for all mobile platforms) this step is still mandatory. 
```

File: Cookbook/Technical-Documents/External-libraries.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+```ruby
+def shared_pods
+	pod 'FBSDKLoginKit', '~> 5.4.1'
+	pod 'Mixpanel-swift', '~> 2.5'
+	pod 'AppsFlyerFramework'
+	pod 'SnowplowTracker', '~> 1.1'
+```
+
+If the framework is specific to particuar apps then add it the relevant targets only.
```
Comment (konrad-em) [inline Cookbook/Technical-Documents/External-libraries.md]:
```suggestion
If the framework is specific to particular apps then add it the relevant targets only.
```

**Review by konrad-em — APPROVED** [review]
Well  done! Found two typos, otherwise good to go 🚢

**Review by AnnKatF — COMMENTED** [review]
This is great!
Thanks for putting it in this repo.
Just the link to the cookbook is missing 😅
Other than that, is good to go 🚀

**Comment by viorel15** [comment]
@AnnKatF Thank you for spotting that. I had updated the cookbook.
As soon as this PR is merged I will also update the main Babylon project PR https://github.com/babylonhealth/babylon-ios/pull/10028 to point to the doc from playbook.

**Review by AnnKatF — APPROVED** [review]
LGTM 👍

**Pull request merged**